### PR TITLE
Akka 2.5.12

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: blog/_posts/2018-04-13-akka-2.5.12-released.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- Behaviors.immutablePartial -> Behaviors.receivePartial
+- Behaviors.onSignal -> Behaviors.receiveSignal
+- Behaviors.immutable ... onSignal -> Behaviors.receive ... receiveSignal
+- Behaviors.mutable -> Behaviors.setup
+- Behaviors.MutableBehavior -> Behaviors.MutableBehavior (top level class)
+- ExtensibleBehavior. receiveMessage -> ExtensibleBehavior. receive
+
+New:
+
+- Behaviors.receiveMessage
```
Comment (ktoso) [inline blog/_posts/2018-04-13-akka-2.5.12-released.md]: Add mini explanation? `(similar to Behaviors.receive, but handling only the message, no context)`

File: _config.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ defaults:
     # note that changing this requires a jekyll restart to be picked up if serving locally
     # versions.json will be updated as long as the changes here are patch versions, for minor and major it needs to
     # be updated manually
-    current_akka_version: 2.5.11
+    current_akka_version: 2.5.12
     current_akka_http_version: 10.1.0
```
Comment (johanandren) [inline _config.yml]: Can you update this one when you are anyways at it, 10.1.1 is the latest.

File: blog/_posts/2018-04-13-akka-2.5.12-released.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+author: Patrik Nordwall
+short: We are happy to announce the 2.5.12 patch release of Akka 2.5
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. This release mostly contains small fixes across the board
+and continued work on Akka Typed. Some notable improvements and bug fixes in 2.5.11 are:
```
Comment (johanandren) [inline blog/_posts/2018-04-13-akka-2.5.12-released.md]: in 2.5.12

File: blog/_posts/2018-04-13-akka-2.5.12-released.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Renames:
+
+- Behaviors.immutable -> Behaviors.receive
+- Behaviors.immutablePartial -> Behaviors.receivePartial
+- Behaviors.onSignal -> Behaviors.receiveSignal
+- Behaviors.immutable ... onSignal -> Behaviors.receive ... receiveSignal
+- Behaviors.mutable -> Behaviors.setup
+- Behaviors.MutableBehavior -> Behaviors.MutableBehavior (top level class)
+- ExtensibleBehavior. receiveMessage -> ExtensibleBehavior. receive
```
Comment (johanandren) [inline blog/_posts/2018-04-13-akka-2.5.12-released.md]: Mark these as code perhaps?

File: blog/_posts/2018-04-13-akka-2.5.12-released.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. This release mostly contains small fixes across the board
+and continued work on Akka Typed. Some notable improvements and bug fixes in 2.5.11 are:
+
+- Leaving and restarting node with Artery could result in that system messages couldn't be delivered [#24847](https://github.com/akka/akka/issues/24847)
```
Comment (raboof) [inline blog/_posts/2018-04-13-akka-2.5.12-released.md]: "Restarting a node with Artery could prevent system messages from being delivered"?

File: _config.yml (hunk 2 above)
Comment (patriknw) [reply _config.yml]: done

File: blog/_posts/2018-04-13-akka-2.5.12-released.md (hunk 3 above)
Comment (patriknw) [reply blog/_posts/2018-04-13-akka-2.5.12-released.md]: ouch, thanks

File: blog/_posts/2018-04-13-akka-2.5.12-released.md (hunk 4 above)
Comment (patriknw) [reply blog/_posts/2018-04-13-akka-2.5.12-released.md]: done

File: blog/_posts/2018-04-13-akka-2.5.12-released.md (hunk 5 above)
Comment (patriknw) [reply blog/_posts/2018-04-13-akka-2.5.12-released.md]: thanks, that is better

**Comment by patriknw** [comment]
incorporated feedback, thanks

**Pull request merged**